libcudacxx/codegen/generators/compare_and_swap.h

```diff
@@ -154,6 +154,8 @@ static inline _CCCL_DEVICE bool __atomic_compare_exchange_cuda(_Type* __ptr, _Ty
   __proxy_t* __ptr_proxy = reinterpret_cast<__proxy_t*>(__ptr);
   __proxy_t* __exp_proxy = reinterpret_cast<__proxy_t*>(__exp);
   __proxy_t* __des_proxy  = reinterpret_cast<__proxy_t*>(&__des);
+  bool __res = false;
+  if (__cuda_compare_exchange_weak_if_local(__ptr_proxy, __exp_proxy, __des_proxy, &__res)) {{return __res;}}
```

Comment:
probably a n00b question, but why the double braces `{{}}`?

Reply:
Double brackets are an escape for `fmt`, they likely just leaked in. Thanks!